<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoMatching_params</class>
 <widget class="QWidget" name="StereoMatching_params">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 203, 173);</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>259</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Optimization (WTA/SGM)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_optType">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>WTA  disp. thresh.:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_wtaThresh">
      <property name="text">
       <string>5.0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Minimal Disparity:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_minDisparity">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Maximal Disparity:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_maxDisparity">
      <property name="text">
       <string>155</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>SAD window</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_SADwin">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Pre-Filter Cap</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_preFilterCAP">
      <property name="text">
       <string>11</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Uniqueness</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_uniqueness">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Speckle Window</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_speckleWin">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Speckle Range</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_speckleRange">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>HOG block size (mult.of cell)</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>HOG cell size</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEdit_HOGcellSize">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEdit_HOGblockSize">
      <property name="text">
       <string>24</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
